src/webapp/ml_inference_serving_app.py

```diff
@@ -67,22 +70,35 @@ def setup_model_inference_serving_app(cls, *args, **kwargs):
 
     @classmethod
     def list_all_model_in_serving(cls):
-        return cls._model_in_serving.keys()
+        return list(cls._model_in_serving.keys())
 
     @classmethod
-    def load_original_model_from_mlflow_artifact(cls, model_name: str, model_version: str, model_stage: str = "Production"):
+    def set_model_from_mlflow_artifact_origin_flavor(cls, model_name: str, model_version=None, model_stage=None) -> bool:
         """
         load original model from mlflow artifact
         implementation in mlflow_agent, distinguish the origin model flavor by mlflow model registry
+        :param model_stage:
         :param model_name: model name
         :param model_version: model version
         :return: True if model is successfully loaded
         """
 
+        # check mlflow agent setup tracking server uri ready or not
+        if cls._mlflow_agent.mlflow_connect_tracking_server_uri is None:
+            raise ValueError("mlflow tracking server is not provided")
+
+        load_model_args = [model_name]
+        if model_version is not None:
+            load_model_args.append(model_version)
+        if model_stage is not None:
+            load_model_args.append(model_stage)
+
         if not cls._is_model_exist(model_name):
             try:
-                fetched_model = cls._mlflow_agent.load_original_model(model_name, model_version, model_stage)
+                fetched_model = cls._mlflow_agent.load_original_model(*load_model_args)
```

Comment:
To make the invocation of the mlflow_agent load model an ad hoc model info, i.e., model_name + (model_version || model_stage).